bedrock/gigabit/templates/gigabit/apply.html

```diff
@@ -139,8 +139,8 @@ <h1 class="large center">{{ _('Apply') }}</h1>
             {% endtrans %}
             </li>
             <li>
-            {% trans url_gigabit_application_form='https://drive.google.com/file/d/0B9NqWPTRsfNXNjlKT0tMVlcySlhfZmVlQTBINXVrN0FSVlI0/view' %}
-              Prepare your application using our <a href='{{ url_gigabit_application_form }}'>application form</a>.
+            {% trans url_gigabit_application_form='https://drive.google.com/file/d/0B9NqWPTRsfNXNjlKT0tMVlcySlhfZmVlQTBINXVrN0FSVlI0/view', url_faq='http://mzl.la/GCF_FAQ' %}
```

Comment:
Unless there's a reason to use a short URL here (e.g. analytics), I'd suggest we change the `href` to be the destination link instead. If we do need to keep the short URL, we can make it `https://`.